packages/server/graphql/intranetSchema/mutations/connectSocket.ts

```diff
@@ -53,12 +53,11 @@ export default {
       adjustUserCount(userId, orgIds, InvoiceItemType.UNPAUSE_USER).catch(console.log)
       // TODO: re-identify
     }
-    const datesAreOnSameDay = now.toDateString() === lastSeenAt?.toDateString()
+    const datesAreOnSameDay = now.toDateString() === lastSeenAt.toDateString()
     if (!datesAreOnSameDay) {
       await updateUser(
         {
-          inactive: false,
-          lastSeenAt: now
```

Comment:
-1 This is the only place were we update `lastSeenAt`

Reply:
oops nice catch!